<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Advance club - Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="labelSpliter">
          <property name="text">
           <string>Split file freq</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="boxSplitor">
          <property name="suffix">
           <string> words</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushChangeTime">
        <property name="text">
         <string>Shange timing</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidgetLog"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerateAudioList"/>
   <addaction name="actionSplitFileList"/>
   <addaction name="actionAudio_list_file"/>
   <addaction name="actionInpute_audio_folder"/>
   <addaction name="actionOutpute_audio_folder"/>
   <addaction name="actionConvert"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAudio_list_file">
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/text.ico</normaloff>:/icons/text.ico</iconset>
   </property>
   <property name="text">
    <string>Audio list file</string>
   </property>
  </action>
  <action name="actionInpute_audio_folder">
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/music_folder.ico</normaloff>:/icons/music_folder.ico</iconset>
   </property>
   <property name="text">
    <string>Inpute audio folder</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/convert.ico</normaloff>:/icons/convert.ico</iconset>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="actionGenerateAudioList">
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/generate_text_file.ico</normaloff>:/icons/generate_text_file.ico</iconset>
   </property>
   <property name="text">
    <string>Generate audio list</string>
   </property>
  </action>
  <action name="actionSplitFileList">
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/split.ico</normaloff>:/icons/split.ico</iconset>
   </property>
   <property name="text">
    <string>SplitFileList</string>
   </property>
  </action>
  <action name="actionOutpute_audio_folder">
   <property name="icon">
    <iconset resource="Converter.qrc">
     <normaloff>:/icons/outpute_folder.ico</normaloff>:/icons/outpute_folder.ico</iconset>
   </property>
   <property name="text">
    <string>Outpute audio folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Converter.qrc"/>
 </resources>
 <connections/>
</ui>
